### DISCUSSION TITLE
Connect two of those devices with each other

### DISCUSSION THREAD

**Discussion by Linteum** [pschatzmann/ESP32-A2DP#528] [Q&A] [answered]
Hi, 
I am wondering if it would be possible to connect one emitter to one receiver ? 
The goal would be to connect two analog devices with each other.

And if you had any idea of the possible delay created by the transmission, i would be delighted ^^

Thank you for you work :)

**Comment by pschatzmann** [comment, marked as answer]
I am not sure if I understand you correctly.
You can send audio from a A2DP source implemented with this library to another ESP32 A2DP sink, implemented with this library.

In any case the lag from A2DP will be horrible...
